body.tex

```diff
@@ -171,40 +168,50 @@ \subsubsection{Technical Managers and other Leadership Roles}
     \item
-        A series of \emph{epics} describe major pieces of work. Epics are associated with concrete, albeit high-level, deliverables in the \gls{shape} of the above milestones, and have specific resource loads (staff assignments), start dates, and durations.
+        A visualization of progress on work done towards achieving milestones is captured in Smartsheet, which directly tracks progress by rolling up issues that are completed inside of \gls{JIRA} epics that work together to deliver a given milestone.      
 \end{enumerate}
 
-Milestones are allocated to one of four levels, defined as follows:
+Milestones are allocated to one of three levels, defined as follows:
 
 \begin{description}
 \item[Level 1]
 These are at the full observatory level and owned by the Directors Office. 
-An example is the completion of an annual Data Release.
+Examples are the completion of a Data Preview, the beginning of nightly observations for the full survey, or the delivery of an annual Data Release. 
+Level 1 milestones are achieved by the culmination of effort defined by a set of Level 2 and Level 3 milestones. 
+Level 1 milestones are reported to the agencies.
 \item[Level 2]
 These reflect effort within a Department and are owned by an Associate Director, or are cross-Department commitments. 
 As such, they must be defined in consultation with the Director's Office.
+Level 2 milestones are achieved by the culmination of effort defined by a set of Level 3 milestones.
+Some Level 2 milestones may be reported to the agencies as defined by the annual POP. 
 \item[Level 3]
 These are internal to a particular Department and assigned to a team and can therefore be specified by a single team lead.
 \end{description}
 
-Some of these milestones are exposed to external reviewers (usually reserved for Level 1 milestones): it is vital that these be delivered on time and to specification.
-Low-level (3 and beyond) milestones are defined for use within Departments, but even here properly adhering to the plan is vital: your colleagues in other teams will use these milestones to align their schedules with yours, so they rely on you to be accurate.
+Some of these milestones are exposed to external reviewers (usually reserved for Level 1 milestones): it is vital that these be delivered on time and to specification. 
+Level 1 and 2 milestones are under change control once they are defined and described in a \gls{JIRA} Milestone issue.  
+Level 3 milestones are defined for use within Departments and not required to go under project change control, but properly adhering to the plan is vital: your colleagues in other teams will use these milestones to align their schedules with yours, so they rely on you to be accurate.
 
-Epics should help achieve milestones i.e. they may be blocking issues on the milestones.
+Epics should work to achieve milestones i.e. they may be blocking issues on the milestones.
```

Comment:
are we making them blocking issues though? you are doing this in smartsheet now ..